git-publish

```diff
@@ -449,14 +449,25 @@ def menu_select(menu):
             continue
         return a
 
+def edit_content(content, suffix):
+    fd, tmpfile = tempfile.mkstemp(text=True, suffix=suffix)
+    fd_opened = os.fdopen(fd, 'wb')
+    fd_opened.write(content)
+    fd_opened.close()
+    edit(tmpfile)
+    fd_opened = open(tmpfile, "rb")
+    new_content = fd_opened.read()
+    fd_opened.close()
+    if tmpfile:
+        os.unlink(tmpfile)
+    return new_content
+
 def edit_email_list(cc_list):
-    tmpfile = tempfile.NamedTemporaryFile(mode='wb', suffix='.txt')
-    tmpfile.write(os.linesep.join(cc_list).encode(ENCODING))
-    tmpfile.flush()
-    edit(tmpfile.name)
+    new_content_bytes = edit_content(os.linesep.join(cc_list).encode('utf-8-sig'), '.txt')
```

Comment:
Why is ENCODING replaced by 'utf-8-sig'?

Reply:
utf-8-sig can support by windows notepad.exe, utf-8 will treat as ANSI encoding.
And for ANSI encoding, some characters(author names) may not recognized by editor